user-data/main.tf

```diff
@@ -48,6 +65,21 @@ EOF
 
     content = "${module.teleport_bootstrap_script.teleport_bootstrap_script}"
   }
+
+  part {
+    content_type = "text/cloud-config"
+
+    content = <<EOF
+#cloud-config
+package_upgrade: true
```

Comment:
the `package_upgrade: true` instruction is already there: https://github.com/skyscrapers/terraform-ecs/blob/master/user-data/main.tf#L7